Mage.Server/src/main/java/mage/server/ChatManager.java

```diff
@@ -157,10 +158,9 @@ public void broadcast(UUID chatId, String userName, String message, MessageColor
                     }
 
                     userMessages.put(userName, message);
-                }
 
-                if (messageType == MessageType.TALK) {
-                    if (user != null) {
```

Comment:
That check is already done, this line is inside the scope of the isPresent() on line#120